SSLChecker/SSLChecker/main.py

```diff
@@ -0,0 +1,227 @@
+import json
+import logging
+from typing import Tuple, Mapping
+from time import process_time
+from urllib.parse import urlparse
+
+import azure.functions as func
+
+from ..sharedcode import shared_dns
+from ..sharedcode import scanner
+from ..sharedcode import results
+from ..sharedcode.errors import (InvalidRequest, DNSError, InvalidConfig,
+                                 ConnectionError, InvalidFQDN)
+
+external_dns, internal_dns = shared_dns.get_dns_options()
+
+dnsview = {"external": external_dns,
+           "internal": internal_dns}
+
+# Valid scan types
+VALID_SCAN_TYPES = ['policy', 'full']
+
+ERROR_MSG_MISSING_PARAMETERS = \
+    ("Please pass three parameters in the URI: "
+     "valid scan type: policy or full, valid DNS view: internal or external, "
+     "and a valid DNS domain name. For example: "
+     "https://sslchecker.metlife.com/api/full/external/www.google.com")

```

Comment:
This is ok I think since its an example, but sslchecker.exampleco.com may be better in the future @gattjoe

Reply:
Fixed in another version.